app/view_objects/articles/social_image.rb

```diff
@@ -3,8 +3,10 @@ class SocialImage
     include Rails.application.routes.url_helpers
     include CloudinaryHelper
 
-    def initialize(article)
+    def initialize(article, **options)
       @article = article
+      @height = options[:height] || 420
```

Comment:
I just noticed, this is 420, but on line 18, it was 500. Intentional change?

Reply:
Also, does it work correctly when you give it a number rather than a string? Looks like they were strings initially, might want to check what `cl_image_path` expects those arguments to be!

Reply:
OH! Good catch. I should have changed that when I moved the logic over. 🤦‍♀️

Reply:
👍 let me know once it's updated, I'll give it another review!

Reply:
I'm trying to test out the strings vs. numbers right now, but I have already made the other change - just need to push it up! 😄